### DISCUSSION TITLE
Next.js App 'ready'-ing server but never actually starting it

### DISCUSSION THREAD

**Discussion by Kriti-Negi** [vercel/next.js#46174] [Help] [answered]
## What I did
I ran the following commands:
```
npx create-next-app nextjs-blog --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter"
cd nextjs-blog
npm run dev
```
Please note that for all the options (ex, do you want typescript) I selected 'no'

## The output
```
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
npm notice 
npm notice New minor version of npm available! 9.3.1 -> 9.5.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.5.0
npm notice Run npm install -g npm@9.5.0 to update!
npm notice 
```

## Problem
Sever for the Next.js app is 'readying' itself and closing almost immediately. The default website is not appearing on localhost:3000

## What I have done
I have tried almost every solution I could think of, including updating my npm, restarting my computer, deleting and re-installing node modules, etc....

## System Info
I'm on a macOS Catalina version 10.15.2

### File structure looks as follows:
<img width="177" alt="Screen Shot 2023-02-20 at 7 44 38 PM" src="https://user-images.githubusercontent.com/66047981/220234295-59caa0d8-dd79-427f-99c7-daaef6d489a6.png">

### package.json file looks as follows

```
{
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start"
  },
  "dependencies": {
    "next": "latest",
    "react": "18.2.0",
    "react-dom": "18.2.0"
  }
}
```

Let me know if I need to give any more information.

**Comment by icyJoseph** [comment, marked as answer]
I've seen this before as a SEGMENTATION FAULT for people with older MacOS versions. 

Could you try the recommended approach here: https://github.com/vercel/next.js/discussions/43436#discussioncomment-4567449

Basically, downgrade to next 12, and remove usage of @next/font

**Comment by Kriti-Negi** [comment]
Thank you =) 
It worked! (I only needed to downgrade to 12, as I did not have any @next/font package).

Do you know of any way I can still use Next.js 13?

**Reply by icyJoseph** [reply]
I wasn't sure until now, but now I wonder if you should open a bug, to at least try to get an statement of what OS' can get support. I wonder if it has to do with some architecture changes going back to Mac Catalina. 

Better to raise it as an issue. 

Include your `next info` output, and state how just downgrading to 12 fixes it. The core maintainers should be able to recall what might have causes this, and potentially give a way forward.
